storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -268,15 +265,6 @@ public void expire(Protos.OfferID key, Protos.Offer val) {
         }
     );
 
-    Number lruCacheSize = Optional.fromNullable((Number) mesosStormConf.get(CONF_MESOS_OFFER_LRU_CACHE_SIZE)).or(1000);
```

Comment:
`CONF_MESOS_OFFER_LRU_CACHE_SIZE` is no longer needed, so please remove that too.